pkg/retry/retry.go

```diff
@@ -69,7 +69,12 @@ func isRetryable(err error) bool {
 		}
 		return isRetryable(e.Err)
 	case syscall.Errno:
-		return e != syscall.ECONNREFUSED
+		switch e {
+		case syscall.ECONNREFUSED, syscall.EINTR, syscall.ERESTART, syscall.EAGAIN, syscall.EBUSY, syscall.ENETDOWN, syscall.ENETUNREACH, syscall.ENETRESET, syscall.ECONNABORTED, syscall.ECONNRESET, syscall.ETIMEDOUT, syscall.EHOSTDOWN, syscall.EHOSTUNREACH:
```

Comment:
This looks about right; maybe still too broad (why EINTR? What is ERESTART?) but clearly better-targeted than the original.

(Please highlight changes like the ECONNREFUSED behavior in the PR description next time.)

Reply:
I was not sure if either of them could get triggered while doing a long network operation.  Basically I read through all possible errnos and attempted to find ones that might make sense.

Reply:
Updated the PR with more information on the change.

@QiWang19 PTAL